### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by aprasolova**
- go fmt's
- make `NeoScanBalancer.GetBalance()` public
- new type for `sendtoaddress` response
- make `GetInvocationScript()` public

### CODE REVIEW

File: pkg/rpc/types.go [hunk 1]
```diff
@@ -73,3 +75,48 @@ type response struct {
...
+	responseHeader
+	Error  *Error `json:"error"`
+	Result *TxResponse
+}
+
+// struct represents verbose output of `getrawtransaction` RPC call
+type GetRawTxResponse struct {
+	responseHeader
+	Error *Error `json:"error"`
+	Result *RawTxResponse `json: "result"`
```
Comment (im-kulikov) [inline pkg/rpc/types.go]: gofmt

Comment (aprasolova) [reply pkg/rpc/types.go]: done

**Comment by im-kulikov** [comment]
@anthdm thx

**Comment by hal0x2328** [comment]
Please reward aprasolova 100 points.

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/144dc06fbc54dd84165af08ddd9c01e8a720ca1bb05471dfd1af1503160f245f

**Pull request merged**